services/userService.js

```diff
@@ -41,7 +41,11 @@ const signIn = async (email, password) => {
   const token = jwt.sign({ userId: user.id }, process.env.SECRET_KEY, {
     expiresIn: '5d',
   });
-  return token;
+
+  const resUser = {};
+  resUser.token = token;
+  resUser.userLastName = user.userLastName;
+  return resUser;
```

Comment:
```suggestion
  return {
    token : token,
    userLastName = user.userLastName
  }
```
별도의 변수 및 빈 객체 선언 대신 곧바로 리턴할 수 있도록 해주세요~!